resources/views/admin/page/index.blade.php

```diff
@@ -142,7 +142,7 @@
             <div class="albums second">
                 <div class="tittle-head">
                     <h3 class="tittle">Discover <span class="new">View</span></h3>
-                    <a href="index.html">
+                    <a href="{{route('radio')}}">
```

Comment:
```suggestion
                    <a href="{{ route('radio')}} ">
```
bổ sung space sau `{{` và trước `}}` nhé em